domain/src/test/java/org/oppia/android/domain/topic/StoryProgressTestHelperTest.kt

```diff
@@ -491,6 +498,61 @@ class StoryProgressTestHelperTest {
     assertThat(storySummary.chapterList[1].chapterPlayState).isEqualTo(ChapterPlayState.COMPLETED)
   }
 
+  @Test
+  fun testProgressTestHelper_markFullProgressForAllTopics_getOngoingTopicListIsEmpty() {
+    storyProgressTestHelper.markFullProgressForAllTopics(
+      profileId,
+      /* timestampOlderThanAWeek= */ false
+    )
+    testCoroutineDispatchers.runCurrent()
+
+    topicController.getOngoingTopicList(
+      profileId
+    ).toLiveData().observeForever(mockOngoingTopicListObserver)
+    testCoroutineDispatchers.runCurrent()
+
+    verifyGetOngoingTopicListSucceeded()
+
+    val ongoingTopicList = ongoingTopicListResultCaptor.value.getOrThrow()
+    assertThat(ongoingTopicList.topicList.size).isEqualTo(0)
+  }
+
+  @Test
+  fun testProgressTestHelper_markFullProgressForAllTopics_getCompletedStoryListIsCorrect() {
+    storyProgressTestHelper.markFullProgressForAllTopics(
+      profileId,
+      /* timestampOlderThanAWeek= */ false
+    )
+    testCoroutineDispatchers.runCurrent()
+
+    topicController.getCompletedStoryList(profileId).toLiveData()
+      .observeForever(mockCompletedStoryListObserver)
+    testCoroutineDispatchers.runCurrent()
+
+    verifyGetCompletedStoryListSucceeded()
+
+    val completedStoryList = completedStoryListResultCaptor.value.getOrThrow()
+    assertThat(completedStoryList.completedStoryList.size).isEqualTo(6)
+  }
+
+  @Test
+  fun testProgressTestHelper_markFullProgressForAllTopics_getTopicListIsCorrect() {
+    storyProgressTestHelper.markFullProgressForAllTopics(
+      profileId,
+      /* timestampOlderThanAWeek= */ false
```

Comment:
Please also add tests for the two versions of the timestamp parameter if it's going to be used to make sure it does the right thing (if it won't be used, suggest just removing it).

Reply:
Removed the timestamp since the time does not impact lesson progress